uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -387,6 +394,48 @@ public void setMqttTimeoutSec(int mqttTimeoutSec) {
         log.info("MQTT timeout set to {} second(s)", mqttTimeoutSec);
     }
 
+    /**
+     * Sets MQTT content type to transmit.
+     *
+     * @param contentType MQTT content type to transmit
+     */
+    @And("I set MQTT 'content type' to {string} to transmit")
```

Comment:
Please use something similar to 
    @And("I set MQTT publish 'retain' flag to {booleanValue}")

Reply:
The step was renamed in 41edf65c896419d5e3fc6a8684a8ffd6126319b9